src/app/components/core/scheduler/update-schedule/update-schedule.component.ts

```diff
@@ -168,14 +168,21 @@ export class UpdateScheduleComponent implements OnInit, OnChanges {
     }
   }
 
+  getRepeatTime() {
+    if (this.selectedScheduleTypeIndex === 1 || this.selectedScheduleTypeIndex === 4) {
```

Comment:
better to keep `**!** (selectedScheduleTypeIndex == 2 || selectedScheduleTypeIndex == 3) to keep same as in html template.